operator/pkg/ciliumendpointslice/ceptocesmap.go

```diff
@@ -167,6 +167,10 @@ func NewCESName(name string) CESName {
 	return CESName(resource.Key{Name: name})
 }
 
+func NewCESNamespacedName(name string, namespace string) CESName {
```

Comment:
I believe this introduces regression. It will be probably easier to discuss on Slack, so I've started a thread: https://cilium.slack.com/archives/C069SNDMLA2/p1726061039032609

tldr: 
After this change, we will be inserting `CESName` with namespace information.
However,  `uniqueCESliceName` only checks by name without namespace information. So won't be able to detect conflicts of names of CES anymore. 

As now we have both `NewCESName` and `NewCESNamespacedName` - we should get rid of the old `NewCESName` to make sure we don't introduce regression somewhere. 
We could probably also get rid of `cesData` in `CESToCEPMapping` as it contains CESName -> Namespace mapping as it won't be anymore needed since CESName contains namespace information.

Reply:
You're right, we changed it now to ensure that the information about namespace isn't taken into account where it isn't needed. resource.Key containing the namespace is now CESKey. CESName is the alias for string, used in contexts, where only the name of CES is needed (like in functions from ceptocesmap.go)